friture/levels.py

```diff
@@ -136,6 +140,7 @@ def handle_new_data(self, floatdata):
         self.old_rms = value_rms
 
         self.level_view_model.level_data.level_rms = 10. * np.log10(value_rms + 0. * 1e-80)
+        self.level_view_model.level_data.level_spl = self.level_view_model.level_data.level_rms + 94.0 + self.calibration
```

Comment:
why 94 ? If this has a specific meaning, could you please create define a variable and name it ? Thanks!